cdap-ui/app/cdap/components/DataPrep/DataPrepTable/index.js

```diff
@@ -188,6 +192,104 @@ export default class DataPrepTable extends Component {
+                          <span>
+                            {head.name}
+                          </span>
+                        </span>
+                      :
+                        <div className="warning-container-wrapper float-xs-left">
+                          <TextboxOnValium
+                            onChange={this.handleSaveEditedColumnName.bind(this, index)}
+                            value={head.name}
+                            onWarning={this.showWarningMessage.bind(this, index)}
+                          />
+                          {
+                            head.showWarning ?
+                              <WarningContainer
+                                message={T.translate('features.DataPrep.DataPrepTable.columnEditWarningMessage')}
+                              >
+                                <div className="warning-btns-container">
+                                  <div
+                                    className="btn btn-primary"
+                                    onClick={this.handleSaveEditedColumnName.bind(this, index, head.editedColumnName, false)}
+                                  >
+                                    {T.translate('features.DataPrep.Directives.apply')}
+                                  </div>
+                                  <div
+                                    className="btn"
+                                    onClick={this.handleSaveEditedColumnName.bind(this, index, head.name, true)}
+                                  >
+                                    {T.translate('features.DataPrep.Directives.cancel')}
+                                  </div>
+                                </div>
+                              </WarningContainer>
+                            :
+                              null
+                          }
+                        </div>
+                    }
+                    <span className="float-xs-right directives-dropdown-button">
+                      <ColumnActionsDropdown
+                        column={head.name}
+                        dropdownOpened={this.columnDropdownOpened}
+                      />
+                    </span>
+                    <span
+                      onClick={this.toggleColumnSelect.bind(this, head.name)}
+                      className={classnames('float-xs-right fa column-header-checkbox', {
+                        'fa-square-o': !this.columnIsSelected(head.name),
+                        'fa-check-square': this.columnIsSelected(head.name)
+                      })}
+                    />
+                  </div>
+                </th>
+              );
+            })
+          }
+        </thead>
+        <tbody>
+          {
+            data.map((row) => {
+              return (
+                <tr key={shortid.generate()}>
+                  {headers.map((head) => {
+                    return <td key={shortid.generate()}><div>{row[head.name]}</div></td>;
```

Comment:
No. I don't think so. When we show highlighted columns I am not using dataprep table anymore. So this doesn't get rendered at all.